source/agora/network/Manager.d

```diff
@@ -554,31 +554,32 @@ public class NetworkManager
                 if (key == this.config.validator.key_pair.address)
                     continue;
 
-                taskman.runTask
-                ({
+                taskman.runTask(
```

Comment:
We probably shouldnt spawn a task either

Reply:
The reason for spawning a task is to avoid doing all clients serially. Not great but I see the reasoning. Will look into removing it soon.